src/@ionic-native/plugins/base64/index.ts

```diff
@@ -0,0 +1,47 @@
+import { Injectable } from '@angular/core';
+import { Plugin, Cordova, IonicNativePlugin } from '@ionic-native/core';
+
+/**
+ * @beta
+ * @name Base64
+ * @description
+ * This Plugin is used to encode base64 of any file, it uses js code for iOS, but in case of android it uses native code to handle android versions lower than v.3
+ *
+ * @usage
+ * ```typescript
+ * import { Base64 } from '@ionic-native/base64';
+ *
+ * constructor(private base64: Base64) { }
+ *
+ * ...
+ *
+ * let filePath: string = 'file:///...';
+ * this.base64.encodeFile(filePath).then((base64File: string) => {
+ *   console.log(base64File);
+ * }, (err) => {
+ *   console.log(err);
+ * });
+ *
+ * ```
+ */
+@Plugin({
+  pluginName: 'Base64',
+  plugin: 'com-badrit-base64',
+  pluginRef: 'window.plugins.Base64',
+  repo: 'https://github.com/hazemhagrass/phonegap-base64',
+  install: 'ionic cordova plugin add com-badrit-base64',
```

Comment:
This `install` line is not needed since the plugin doesn't require any variables.